composer.json

```diff
@@ -48,16 +47,15 @@
         "composer/package-versions-deprecated": "^1.11.99.5",
         "laminas/laminas-cli": "^1.8",
         "laminas/laminas-coding-standard": "~2.5.0",
-        "laminas/laminas-container-config-test": "^0.8",
-        "laminas/laminas-dependency-plugin": "^2.2",
+        "laminas/laminas-container-config-test": "dev-qa/service-manager-v4",
         "lctrs/psalm-psr-container-plugin": "^1.9",
         "mikey179/vfsstream": "^1.6.11@alpha",
-        "ocramius/proxy-manager": "^2.14.1",
+        "friendsofphp/proxy-manager-lts": "^1",
```

Comment:
Should be extracted to a dedicated satellite package which decorates the container to add proxy functionality.
Is that something which can be easily implemented @Ocramius?

Reply:
https://github.com/laminas/laminas-servicemanager/issues/186